backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Mocks/Altinn.AccessManagement.UI.Mocks.csproj

```diff
@@ -12,6 +12,15 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="Data\SingleRight\CreateDelegation\3225.json" />
```

Comment:
What are these for?

Reply:
Hmm, I think they were added automatically by visual studio, but I don't think we really need to specify them all independently like this so I'll just remove it